src/runner/bootstrapper.ts

```diff
@@ -186,6 +188,28 @@ export default class Bootstrapper {
         return compiler.getTests();
     }
 
+    private _assertGlobalHooks (): void {
+        if (this.hooks?.fixture?.before)
```

Comment:
Let's try to reduce elvis operator count.
```js
const globalFixtureHook = this.hooks?fixture;

if (!globalFixtureHook)
    return;

if (globalFixtureHook.before)
   assertType(is.function, 'globalBefore', 'The fixture.globalBefore hook', globalFixtureHook.before);
...
```

Reply:
I added only this condition:
```javascript
if (!this.hooks)
    return;
```
because other hooks will be added there too